### PR TITLE
docs: update document directory

### PR DESCRIPTION
**Pull request by liangliang4ward**
### What this PR does / why we need it:

update `plugin-develop` document directory

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

File: docs/zh/latest/plugin-develop.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ title: 插件开发
 
 - [目录](#目录)
 - [检查外部依赖](#检查外部依赖)
-- [插件命名与配置](#插件命名与配置)
+- [插件命名，优先级和其他](#插件命名，优先级和其他)
```
Comment (zaunist) [inline docs/zh/latest/plugin-develop.md:28]: Why we need to change this description?  Using commas in titles seems a little strange...

Comment (leslie-tsang) [inline docs/zh/latest/plugin-develop.md:28]: I'd like to know why we changed the title, BTW, the link doesn't work in GitHub Markdown.

Comment (tzssangglass) [reply docs/zh/latest/plugin-develop.md:28]: Because the following title is used in `插件命名，优先级和其他`

Comment (tzssangglass) [reply docs/zh/latest/plugin-develop.md:28]: yes, In fact, I think it would be better to remove the `，` both in nav and docs ……

**Pull request merged**